default/python/universe_generation/starsystems.py

```diff
@@ -53,21 +53,21 @@ def name_planets(system):
     name + 'asteroid belt' (localized).
     """
     planet_number = 1
+    orbit_number = 1
     # iterate over all planets in the system
     for planet in fo.sys_get_planets(system):
         # use different naming methods for "normal" planets and asteroid belts
         if fo.planet_get_type(planet) == fo.planetType.asteroids:
             # get localized text from stringtable
-            name = fo.user_string("PL_ASTEROID_BELT_OF_SYSTEM")
-            # %1% parameter in the localized string is the system name
-            name = name.replace("%1%", fo.get_name(system))
+            name = fo.get_name(system) + " " + fo.user_string("NEW_ASTEROID_SUFFIX_ORBIT_" + (fo.roman_number(orbit_number)))
```

Comment:
maybe clamp the orbit number to be no bigger than the highest number present in the stringtable, in case more planets are added than 7?